### PR TITLE
Include 0 period fetching

### PR DESCRIPTION
**Pull request by icedTet**
There seems to be a skill issue in fetching 0 periods. attached is code resolving this lack of skill

### CODE REVIEW

File: functions/src/sgyfetch.ts [hunk 1]
```diff
@@ -5,7 +5,7 @@ import {get} from './util/sgyOAuth';
 const SEMESTER = 2; // We are in semester 2
 
 const firestore = admin.firestore();
-const periods = ['1', '2', '3', '4', '5', '6', '7', '8', 'SELF'];
+const periods = ['0','1', '2', '3', '4', '5', '6', '7', '8', 'SELF'];
```
Comment (ky28059) [inline functions/src/sgyfetch.ts:8]: Can you include a space between `'0'` and `'1'`? This looks a bit messy without the space.

**Review by ky28059 — CHANGES_REQUESTED** [review]

File: functions/src/sgyfetch.ts (hunk 1 above)
Comment (icedTet) [reply functions/src/sgyfetch.ts:8]: ... fine

**Comment by icedTet** [comment]
ok @ky28059 here

**Review by ky28059 — APPROVED** [review]
Alright, looks good 👍

**Pull request merged**